setup.cfg

```diff
@@ -1,13 +1,21 @@
+[metadata]
+name = flashcontainer
+version = 0.0.4
+
 [options]
 packages = find:
 package_dir =
     =src
 include_package_data = True
 
+[options.entry_points]
```

Comment:
See above comment. The entry point are in the pyprojects.toml file (@line 42):

[project.scripts]
pargen = "flashcontainer.pargen:pargen_cli"

Reply:
See previous comment.